README.md

```diff
@@ -1 +1,9 @@
- # Documentation
\ No newline at end of file
+# Documentation
+
+**PROJECT DEFINITION**
+
+Our application gives the user theopportunity to use a dropdown menu and choose an ingredient (ex. Gin, Vodka,Cola, Tonic). Based on the decision the application checks a database filledwith cocktails and provides the user with all the possible cocktails with the choseningredient (ex. Gin Tonic, Vodka sour, Cuba Libre). **Extra**
```

Comment:
Please add a line break after each sentence. And remove the title **Extra** from this paragraph.